x/auctions/keeper/grpc_query.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/evmos/v19/utils"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// AuctionInfo returns the current auction information
+func (k Keeper) AuctionInfo(c context.Context, _ *types.QueryCurrentAuctionInfoRequest) (*types.QueryCurrentAuctionInfoResponse, error) {
```

Comment:
Should we return an error if there are no auction because the module is disabled in the param?

Reply:
Yeah good catch, added a check.